### PR TITLE
complete GET /api/topics ncnews-3

### PR DESCRIPTION
**Pull request by LukeBooker**
Completed GET /api/topics, set up test suite and added to error README - ncnews-3

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const db = require("../db/connection");
+
+afterAll(() => {
+  db.end();
```
Comment (DCrawley94) [inline __tests__/app.test.js:8]: Remember as we have opening this callback up with curly brackets it is no longer implicitly returning db.end() - we need to explicitly tell it to!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              expect.objectContaining({
+                description: expect.any(String),
+                slug: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+    test("Status:404, path not found", () => {
+      return request(app)
```
Comment (DCrawley94) [inline __tests__/app.test.js:32]: This is a good test to have for something like /api/topic/:topic_id however I'd advise testing something like /api/not_an_endpoint to test your invalid path handling :)

**Review by DCrawley94 — COMMENTED** [review]
Looking good so far! Have a look at the comments and let me know if you need clarification :)

File: __tests__/app.test.js (hunk 1 above)
Comment (LukeBooker) [reply __tests__/app.test.js:8]: Great, thanks! Just read the Supertest notes again. Lost the brackets and changed to afterAll(() => db.end());

**Pull request merged**